### PR TITLE
Generate Supplemental Figure 2

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
# Generate Supplemental Figure 2

In this PR, I generate the supplemental figure (currently #2) for the NF1 manuscript. I also added into the data analysis repo an environment for figure generation and a new module for figures.

### CODE REVIEW

**Comment by jenna-tomkinson** [comment]
supp_figure_2_plate4_platemaps

Supplemental Figure 2: Plate maps for siRNA constructs and genotype layouts

(A) Plate map layout of the three siRNA constructs and dose on a 96-well plate. (B) Plate map layout of genotype on the same 96-well plate.

File: 4.figures/figures/supplementary/supp_figure_2_plate4_platemaps.png
Comment (MattsonCam) [inline 4.figures/figures/supplementary/supp_figure_2_plate4_platemaps.png:1]: This looks great! Could also consider increasing the sizes of the shapes (triangles, pluses) and the the tick labels (E and 11), although maybe not necessary

**Review by MattsonCam — APPROVED** [review]
Nice looking figure, good job @jenna-tomkinson!

File: 4.figures/figures/supplementary/supp_figure_2_plate4_platemaps.png
Comment (jenna-tomkinson) [reply 4.figures/figures/supplementary/supp_figure_2_plate4_platemaps.png:1]: I will implement this now! Thanks for the suggestions!

**Comment by jenna-tomkinson** [comment]
@MattsonCam Thanks for the review! I will be merging now!

**Pull request merged**